webapp/channels/src/components/admin_console/permission_schemes_settings/permission_team_scheme_settings/permission_team_scheme_settings.tsx

```diff
@@ -650,14 +650,14 @@ export default class PermissionTeamSchemeSettings extends React.PureComponent<Pr
                                             defaultMessage='Scheme Name:'
                                         />
                                     </label>
-                                    <LocalizedInput
-                                        id='scheme-name'
+                                    <input
                                         className='form-control'
+                                        disabled={this.props.isDisabled}
+                                        id='scheme-name'
+                                        placeholder={this.props.intl.formatMessage({id: 'admin.permissions.teamScheme.schemeNamePlaceholder', defaultMessage: 'Scheme Name'})}
```

Comment:
Any reason for the reorder? Not complaining, but it makes the diff harder to read.

Reply:
I just put them in alphabetic order. Imo it's more readable this way. But I don't mind preserving the old order either.

Reply:
up to @M-ZubairAhmed

Reply:
I would have preferred to not change order just because of the diff changes but it's fine to leave as is for now